### ISSUE TITLE
spilled teacup sign of the wrist

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#10732]
**Preferred term label:**
spilled teacup sign of the wrist

**Synonyms**

**Definition (free text, please give PubMed ID)**
Lateral wrist x-ray showing proximal dorsal displacement of the capitate and volar displacement of the dislocated lunate in a flexed position

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Perilunate dislocation (PLD)

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
@MickeySegal 
Adding new term: 
- ID: HP:6001395
- label: Spilled teacup sign of the wrist
- parents: [Abnormal lunate bone morphology (HP:0004248)](https://hpo.jax.org/app/browse/term/HP:0004248)
- pmid: PMID:37511801|PMID:23616330
- def: Dislocation of the lunate bone, generally as a result of trauma. Volar displacement of the lunate is said to result a tipped teacup.
- comment: Lateral wrist x-ray showing proximal dorsal displacement of the capitate and volar displacement of the dislocated lunate in a flexed position.
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/10732
- synonyms:
